security.md

```diff
@@ -90,6 +92,32 @@ All security advisories made for Foreman are listed below with their correspondi
 
 ### Disclosure details
 
+#### <a id="2024-7923"></a>CVE-2024-7923: Authentication bypass in Pulpcore
+
+An authentication bypass vulnerability has been identified in Pulpcore when
+deployed by the Foreman Installer with Gunicorn versions prior to 22.0.
+
+This issue arises from the way Apache is configured to do certificate auth and
+pass this information to the Gunicorn backend,
+without unsetting all headers coming from a possibly malicious client.
+
+* Affects Katello 4.0.0 and higher
+* Fix released in Foreman 3.10.1, 3.11.2, 3.12.0
```

Comment:
Note to self: I think 3.12.0 is sufficient, even if it was already released in rc2.